cw-orch-daemon/examples/event-query.rs

```diff
@@ -0,0 +1,24 @@
+use cw_orch_core::environment::DefaultQueriers;
+use cw_orch_core::environment::NodeQuerier;
+use cw_orch_daemon::DaemonBuilder;
+use cw_orch_networks::networks;
+
+const LOCAL_MNEMONIC: &str = "clip hire initial neck maid actor venue client foam budget lock catalog sweet steak waste crater broccoli pipe steak sister coyote moment obvious choose";
```

Comment:
```suggestion
// https://github.com/CosmosContracts/juno/blob/32568dba828ff7783aea8cb5bb4b8b5832888255/docker/test-user.env#L2
const LOCAL_MNEMONIC: &str = "clip hire initial neck maid actor venue client foam budget lock catalog sweet steak waste crater broccoli pipe steak sister coyote moment obvious choose";
```
Would be nice to add reference or comment on random mnemonics, so reviewer(or anyone who steps on it next time in the code) won't check if it's public or you accidentally leaked something haha

Reply:
Hmm, good idea !